migrations/cockroach/V1.19__idp_configs.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE management.idp_configs DROP COLUMN logo_src;
+ALTER TABLE adminapi.idp_configs DROP COLUMN logo_src;
+ALTER TABLE auth.idp_configs DROP COLUMN logo_src;
```

Comment:
i believe these will fail as sql_safe_update is on...
@adlerhurst does it work with flyway?